client/task-list/tasks/payments/PaymentGatewaySuggestions/components/List/test/list.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { List } from '../List';
+
+jest.mock( '@woocommerce/tracks', () => ( {
+	recordEvent: jest.fn(),
+} ) );
+
+const mockGateway = {
```

Comment:
Not necessary for the time being, but just making a mental note that we may want to create some fixtures to share between these components in the future if we add more tests.